<p>
    Obsolete deep (<code>&gt;&gt;&gt;</code> or <code>/deep/</code>) combinator should not be used in production code.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
foo /deep/ p {
}

foo >>> p {
}
</pre>

<h2>See</h2>
<ul>
    <li><a href="https://www.w3.org/TR/css-scoping-1/#selectordef-deep">W3C CSS Scoping Module Level 1 Specifications:
        Deep combinator</a></li>
    <li>
        <a href="https://hackernoon.com/the-new-angular-ng-deep-and-the-shadow-piercing-combinators-deep-and-drop-4b088dbe459">The
            New Angular ::ng-deep and the Shadow-Piercing Combinators Drop</a>
    </li>
</ul>
